styles.css

```diff
@@ -107,6 +109,40 @@ a.header-link:hover {
 }
 /* ---header--- */
 
+/* ---main--- */
+main {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	position: relative;
+}
+
+.noescape-cover {
+	width: 100%;
+	/* move main-image top on header */
+	position: relative;
+	/* header exact height: */
+	/* top: -148.672px; */
+	top: -75px;
+}
+
+.noescape-cover img {
+	display: block;
+	max-width: 100%;
+	height: auto;
+}
+
+.image-quote {
+	/* move image-quote top along with main-image */
+	position: relative;
+	top: -50px;
```

Comment:
```suggestion
```

Reply:
I've added margins both top and rights also for this span, since I like how we got some air on top under the main-image and also we need some before main content (Hell Diary)